spec/support/webmock.rb

```diff
@@ -1 +1,3 @@
 require 'webmock/rspec'
+
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
If we don't set this, then the chrome headless driver has its requests blocked. Allowing just localhost seems safe as we won't bombard any actual services with test requests